packages/@office-iss/react-native-win32/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.win32.js

```diff
@@ -347,6 +347,30 @@ const AccessibilityInfo = {
     }
   },
 
+  /**
+   * Post a string to be announced by the screen reader.
+   * - `announcement`: The string announced by the screen reader.
+   * - `options`: An object that configures the reading options.
+   *   - `queue`: The announcement will be queued behind existing announcements. iOS only.
+   */
+  announceForAccessibilityWithOptions(
+    announcement: string,
+    options: {queue?: boolean},
+  ): void {
+    if (Platform.OS === 'android' || Platform.OS === 'win32') {
```

Comment:
>'android' [](http://example.com/codeflow?start=24&length=9)

(nit) aren't there existing string consts for those values somewhere that can be reused?

Reply:
Typescript and Flow both often use string literals in place of enums, where the type is a set of allowed literals.